utils/validator_info.go

```diff
@@ -61,7 +61,7 @@ func GetValidatorsGeoIPInfo(rpc string, geoIPConfig *cfg.EndpointService) (model
 		validatorGeoIPInfo := setValidatorInfo(validator)
 		indexPeerWithSameID := PeerIndex(peers, validator.NodeID)
 		if indexPeerWithSameID >= 0 {
-			err = setGeoIPInfo(validatorGeoIPInfo, peers[indexPeerWithSameID].IP, geoIPConfig)
+			_ = setGeoIPInfo(validatorGeoIPInfo, peers[indexPeerWithSameID].IP, geoIPConfig)
```

Comment:
Not a big fan of shadowing errors. It will be hard to notice if sth goes wrong.
Please add at least a warn log entry or sth similar so we know if it breaks.

Reply:
ok, fixed in the last commit